src/main/java/sx/blah/discord/api/ClientBuilder.java

```diff
@@ -63,207 +63,210 @@
 	private long eventThreadTimeout = 60L;
 	private TimeUnit eventThreadTimeoutUnit = TimeUnit.SECONDS;
 	private int overflowCapacity = 128;
-
 	private StatusType status;
 	private String playingText;
 	private String streamUrl;
-
 	//Early registered listeners:
 	private final List<IListener> iListeners = new ArrayList<>();
 	private final List<Object> listeners = new ArrayList<>();
 	private final List<Class<?>> listenerClasses = new ArrayList<>();
 
 	/**
-	 * Provides the login info for the client.
+	 * Configures the bot token used for authentication with Discord.
 	 *
 	 * @param token The bot's token.
-	 * @return The instance of the builder.
+	 * @return The builder instance.
```

Comment:
Why not use the same as [EmbedBuilder](https://github.com/austinv11/Discord4J/blob/master/src/main/java/sx/blah/discord/util/EmbedBuilder.java#L103)'s return for this. ("Itself for chaining")